build.gradle

```diff
@@ -45,6 +45,7 @@ if (System.env.REPO_NAME) {
 }
 
 minecraft {
+	accessWidener = file('src/main/resources/screen-handlers.accessWidener.txt')
```

Comment:
drop the txt, I think we should standardise on `.accesswidener` file extension

Reply:
Fixed.